lib/galaxy/tool_util/deps/container_resolvers/mulled.py

```diff
@@ -382,11 +400,18 @@ def __init__(self, app_info=None, namespace="biocontainers", hash_func="v2", aut
         self.auto_install = string_as_bool(auto_install)
 
     def cached_container_description(self, targets, namespace, hash_func, resolution_cache):
-        return docker_cached_container_description(targets, namespace, hash_func, resolution_cache)
+        try:
+            return docker_cached_container_description(targets, namespace, hash_func, resolution_cache)
+        except subprocess.CalledProcessError:
+            # We should only get here if a docker binary is available, but command quits with a non-zero exit code,
+            # e.g if the docker daemon is not available
+            log.exception('An error occured while listing cached docker images. Disabling docker image cache.')
+            self.docker_cli_available = False
```

Comment:
Since there is no method to re-enable `docker_cli_available` apart from a Galaxy server restart, can't we just return None without disabling this?
The docker daemon could simply be restarted by an admin that notices the error in the Galaxy logs.

Reply:
sounds good, will do that